### ISSUE TITLE
Right click not dismissing / hiding an open menu  (Windows only)

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#7403]
## Test plan
1. Launch Brave on Windows
2. Visit Preferences, disable `Hide the menu bar by default`, and confirm the menu shows up
3. Click any menu item, such as the `View` menu and wait for the menu to display
4. Right click any area to the right of the menu and confirm the menu closes

See the GIF below (in this post) for what it looks like when it fails to close

## Original issue description
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Right click doesn't clear menu bar 

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
Brave	0.13.5
rev	7373a01

- Steps to reproduce:
    1. Enable menu bar from settings
    2. Open any menu bar item
    3. Right click on the browser window, menu item is not dismissed 
    
- Actual result:
Menu item is not dismissed on right click 

- Expected result:
Should dismiss menu item when right click/left click

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
![rclick](https://cloud.githubusercontent.com/assets/17010094/23356489/f3a9141a-fcd1-11e6-82ff-a9bc8448c3ed.gif)

- Any related issues:
cc: @NejcZdovc

**Comment by NejcZdovc** [comment]
Correct, right click is not handled correctly. This was introduced with chromuim upgrade, @bbondy fixed it here da801c71. But I think the core problem here is that context menu wrapper is not handled correctly.

You can see that blue box is much bigger then context menu:
![image](https://cloud.githubusercontent.com/assets/9574457/23357637/fd65b94c-fcde-11e6-9543-b7a571567e2b.png)

This is because width and height is set to 100%: https://github.com/brave/browser-laptop/blob/54ac25ad14589492e723db012152adfa341a68d4/less/contextMenu.less#L23-L24

Because of that for example you don't have same behavior if you click or hover on the left/right side of the menu:

![untitled3](https://cloud.githubusercontent.com/assets/9574457/23357506/599206a4-fcde-11e6-9da7-1744c5d1ed55.gif)

cc @bsclifton

**Comment by alexwykoff** [comment]
@luixxiul those labels exist so the generator doesn't add them to the test runs. when i get a block of time i'll update the generator, but this is the SOP for now 😸
